src/posts/dto/create-post.dto.ts

```diff
@@ -1,5 +1,11 @@
+import { IsBoolean, IsNotEmpty } from 'class-validator';
+
 export class CreatePostDto {
```

Comment:
I would suggest to use `PostCreationDTO` instead.

Reply:
Done.